src/include/btmem.h

```diff
@@ -28,11 +28,12 @@
 /* AUTOMATIC FLAG VALUE GENERATION START */
 #define WT_REC_CHECKPOINT 0x01u
 #define WT_REC_EVICT 0x02u
-#define WT_REC_HS 0x04u
-#define WT_REC_IN_MEMORY 0x08u
-#define WT_REC_SCRUB 0x10u
-#define WT_REC_VISIBILITY_ERR 0x20u
-#define WT_REC_VISIBLE_ALL 0x40u
+#define WT_REC_EXPECT_CLEAN 0x04u
+#define WT_REC_HS 0x08u
+#define WT_REC_IN_MEMORY 0x10u
+#define WT_REC_SCRUB 0x20u
+#define WT_REC_VISIBILITY_ERR 0x40u
+#define WT_REC_VISIBLE_ALL 0x80u
```

Comment:
If we have to add another flag here, I think we'll need to deal with it for internal pages as well (i.e., the other places where `WT_REC_VISIBILITY_ERR` is checked.